include/libultraship/classes.h

```diff
@@ -25,6 +25,7 @@
 #include "controller/KeyboardScancodes.h"
 #include "controller/DummyController.h"
 #include "binarytools/BinaryReader.h"
+#include "binarytools/MemoryStream.h"
```

Comment:
```suggestion
#include "binarytools/MemoryStream.h"
#include "binarytools/BinaryWriter.h"
```
Might as well also include BinaryWriter if both of these will be there.